### DISCUSSION TITLE
home page sensor % value (display precision)

### DISCUSSION THREAD

**Discussion by robgough1970** [Blackymas/NSPanel_HA_Blueprint#494] [Q&A] [answered]
Is it possible to display sensor values without decimals? I have a sensor on the home page to show ev battery level. in HA it is configure to not show decimals, so for example currently in HA it is dispayed as 43% but on the panel it is showing as 43.0%. i just think it would make it look neater as i'm not interested in the .0

**Comment by kroon040** [comment]
Same issue here. I have the current power in watts, and I also get 350.0@ intead of the 350W

**Comment by Blackymas** [comment, marked as answer]
We have no influence on what HA hands over to us. We only display the value.
If the value is passed with X.X, it is also displayed like this

**Reply by robgough1970** [reply]
ok thanks , thought i would just ask the question. Understand there will be limitations.

**Comment by edwardtfn** [comment]
It could be possible to implement the new [Sensor display precision](https://www.home-assistant.io/blog/2023/03/01/release-20233/#sensor-display-precision), so you could define the number of decimals per entity directly from Home Assistant.
I would love to see this implemented in this Blueprint as soon as possible, however that will force the minimum HA version requirement from v2022.11.1 to v2023.3.1, which could be an issue right now.

**Comment by edwardtfn** [comment]
I'm closing this discussion as we have released v4.0 (#1057) which uses precision from HA in some components.
Please add a feature request for the missing things.

And thanks a lot for your support.
